docs/developers/02-quick-reference/02-cookbook.md

```diff
@@ -134,10 +157,18 @@ export const WalletComponent = () => {
 
 ### useContract
 
-```js title="src/lib/useContract.js"
+```js title="src/lib/useContract.js" showLineNumbers
 import { useContext } from 'react';
 import { NearContext } from './lib/near-provider';
 
+type ContractConfig = {
```

Comment:
Since this is targeted towards JavaScript codebases for now, we can't have types. This might change in future versions, but for now we cannot have type definitions